packages/vc-verification/test/issuer-verification-test.ts

```diff
@@ -627,6 +627,78 @@ function testSuite() {
       ).to.be.true;
     });
 
+    it('invalid RoleEIP191Jwt should not break verification,', async () => {
```

Comment:
I think it's better to be a bit more specific about what is invalid about the RoleEIP191Jwt. Is it that the `claimType` field is invalid?

Reply:
Yeah, but now I think this should change and If the RoleEIP191Jwt is not valid then it should break.

Reply:
Well I think `breaking` implies that the code isn't functioning as expected, which is not what we want. I think what we want to express is that the `verified` return value should be false

Reply:
Yes, this test should ensure that verification returns true or false regardless if there are invalid among claims. verifyIssuer should not throw